src/_SignAssembly.targets

```diff
@@ -0,0 +1,32 @@
+<Project>
```

Comment:
Is there any good reason to use the underscore in the file name?

Reply:
Same reply as above. It's by "convention" - not a well-described one so - I'm good with naming it whatever match the current solution naming. 

I would usually place msbuild files such as this in a subfolder but I did not want to introduce this for only 1 file.

Since we are now also signing the net6 assembly always, the content of the file could really just go in Directory.Build.targets if this is preferred? 

It is right now being loaded via the `CustomBeforeMicrosoftCommonTargets` prop which makes it possible to evaluate conditions based on properties that gets set in the individual csproj files (so it is loaded after the targets defined in these, directory.build.targets is loaded before)

Reply:
I looked into some other repos to check how signing is managed here. It looks like `SignAssembly` and `AssemblyOriginatorKeyFile` properties are quite enough to do the job.